Cargo.toml

```diff
@@ -135,7 +135,7 @@ api_identity = { path = "api_identity" }
 approx = "0.5.1"
 assert_matches = "1.5.0"
 assert_cmd = "2.0.12"
-async-bb8-diesel = { git = "https://github.com/oxidecomputer/async-bb8-diesel", rev = "da04c087f835a51e0441addb19c5ef4986e1fcf2" }
+async-bb8-diesel = { git = "https://github.com/oxidecomputer/async-bb8-diesel", branch = "error-handling" }
```

Comment:
TODO(me): Before merging, switch this back to the rev of https://github.com/oxidecomputer/async-bb8-diesel/pull/54